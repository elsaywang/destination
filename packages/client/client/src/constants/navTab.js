export const routes = [
    { route: '/', name: 'All' },
    { route: '/destinations', name: 'People-Based' },
    { route: '/deviceBased', name: 'Device-Based' },
    { route: '/cookieBased', name: 'Cookie-Based' },
    { route: '/urlBased', name: 'URL-Based' },
    { route: '/adobeExperienceCloud', name: 'Adobe Experience Cloud' },
];
