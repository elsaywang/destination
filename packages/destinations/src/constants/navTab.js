export const routes = [
    { route: '/destinations/all', name: 'All', types: [] },
    {
        route: '/destinations/adobeExperienceCloud',
        name: 'Adobe Experience Cloud',
        types: ['ANALYTICS'],
    },
    { route: '/destinations/integratedPlatforms', name: 'Integrated Platforms', types: ['S2S'] },
    { route: '/destinations/custom', name: 'Custom', types: ['PUSH', 'ADS'] },
];
