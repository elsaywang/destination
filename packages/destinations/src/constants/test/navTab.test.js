import { routes } from '../navTab';

describe('nav Tab routes constants', () => {
    it('matches snapshot', () => {
        expect(routes).toMatchSnapshot();
    });
});
