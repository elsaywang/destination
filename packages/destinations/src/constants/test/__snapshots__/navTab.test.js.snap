// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`nav Tab routes constants matches snapshot 1`] = `
Array [
  Object {
    "name": "All",
    "route": "/destinations/all",
    "types": Array [],
  },
  Object {
    "name": "Adobe Experience Cloud",
    "route": "/destinations/adobeExperienceCloud",
    "types": Array [
      "ANALYTICS",
    ],
  },
  Object {
    "name": "Integrated Platforms",
    "route": "/destinations/integratedPlatforms",
    "types": Array [
      "S2S",
      "PEOPLE_BASED",
    ],
  },
  Object {
    "name": "Custom",
    "route": "/destinations/custom",
    "types": Array [
      "PUSH",
      "ADS",
    ],
  },
]
`;
